aslam_cv_cameras/src/ncamera-yaml-serialization.cc

```diff
@@ -73,9 +73,10 @@ bool convert<std::shared_ptr<aslam::NCamera> >::decode(const Node& node,
         return true;
       }
       // This call will fail hard if the matrix is not a rotation matrix.
-      aslam::Quaternion q_B_C = aslam::Quaternion(
+      const aslam::Quaternion q_B_C =
+          aslam::Quaternion::fromApproximateRotationMatrix(
```

Comment:
what's the precision here? can we still impose a threshold?

Reply:
yes, there is still a threshold. it's 1e-4 instead of 1e-8.